### ISSUE TITLE
[Accessibility] Poor text contrast on purple buttons (black text on purple background) for the sidebar option

### ISSUE THREAD

**Issue by Pradumnasaraf** [kestra-io/kestra#11249]
### Describe the issue

For the sidebar option with sub-options, the main parent option changes to black (instead of white) on a purple background, making it hard to read. This issue occurs only with the **Administration** and **Blueprints**.

Happening with only light mode.

### Visuals

<img width="290" height="439" alt="Image" src="https://github.com/user-attachments/assets/a8632733-6690-4095-b91c-7cfc50890e2a" />
<img width="290" height="439" alt="Image" src="https://github.com/user-attachments/assets/c95382b9-5901-4b66-ad41-3a68a8451bfb" />

https://jam.dev/c/9dd38140-77d8-48ab-a5c4-b2a3d3e7166c

### Environment

- Kestra Version: develop

---

Similar issue for a different element on the page kestra-io/kestra-ee#5094

**Comment by Piyush-r-bhaskar** [comment]
I observed the Administration part while doing this https://github.com/kestra-io/kestra/pull/11252 and fixed already.

we have a separate issue https://github.com/kestra-io/kestra/issues/11231 for footer one...I will tackle requires changing in EE.
